go/engine/pgp_encrypt.go

```diff
@@ -65,10 +65,7 @@ func (e *PGPEncrypt) SubConsumers() []libkb.UIConsumer {
 // Run starts the engine.
 func (e *PGPEncrypt) Run(ctx *Context) error {
 	// verify valid options based on logged in state:
-	ok, uid, err := IsLoggedIn(e, ctx)
-	if err != nil {
-		return err
-	}
+	ok, uid, _ := IsLoggedIn(e, ctx)
```

Comment:
I need a better comment there. Basically, `IsLoggedIn` will return `true` iff `err == nil`. The `err` is there if you want to make finer-grained decisions about the error.

Reply:
It would probably be OK to bail out there is an error because the app is in some broken state, but the operation can still succeed despite the broken state....